db/migrate/20180102134015_add_external_identifier_to_person.rb

```diff
@@ -0,0 +1,5 @@
+class AddExternalIdentifierToPerson < ActiveRecord::Migration[5.0]
+  def change
+    add_column :people, :external_identifier, :string
```

Comment:
hier moeten we denk ik nog even iets bij zetten voor het aanmaken van `external identifiers`s